packages/cozy-client/src/types.js

```diff
@@ -30,9 +30,42 @@ import { QueryDefinition } from './queries/dsl'
 
 /**
  * @typedef {object} KonnectorsDocument
- * @property {String} slug - slug of konnector
- * @property {ManifestFields} fields - konnector fields
- * @property {Boolean} clientSide - whether the konnector runs on client or not
+ * See https://docs.cozy.io/en/cozy-doctypes/docs/io.cozy.konnectors/
+ *
+ * @property {object} [aggregator] - object containing aggregator data
+ * @property {Array} [categories] - list of categories (default: ['others'])
+ * @property {Boolean} [clientSide] - whether the konnector runs on client or not
+ * @property {Array} [data_types] - list of data types managed by the connector
```

Comment:
You can't do that? `Array.<string>` ? Same for categories and other array

Reply:
You're right after checking JSDoc documentation I've specified the type of each Array by adding `[]` to the type contained in the Array (e.g. String[])